index.css

```diff
@@ -87,3 +87,8 @@ button {
   box-shadow: var(--box-shadow-lg);
   transition: all 0.3s ease;
 }
+@media screen and (max-width: 640px) {
+  .form-section.form-scale {
```

Comment:
Good try, but not quite there yet.

- Look carefully at the solution on Code Monster to see how to implement this styling correctly.